### ISSUE TITLE
VIZ Manga

### ISSUE THREAD

**Issue by loocool2** [tachiyomiorg/tachiyomi-extensions#16373]
### Source name

VIZ Manga

### Source link

https://www.viz.com/vizmanga

### Source language

English

### Other details

First three chapters of all manga are free to read, the rest require a subscription
Simulpub manga have the last three chapters available to read as well.
Different from the Viz Shonen Jump subscription service as per their faq at https://www.viz.com/company-faq#vm
> Subscribe for only $1.99/month to access the VIZ Manga digital library of 10,000+ manga chapters! New subscribers get their first week for free!*
> For only $2.99/month, you can unlock up to 100 chapters every day from the Shonen Jump digital vault of 15,000+ manga chapters! New members get their first week for free!*

Latest can be the recent sort and browse A-Z
https://www.viz.com/read/vizmanga/section/free-chapters

Basically the same structure as the shonen jump section so Viz might be able to be made into a multisrc

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a title with source name.
- [X] I have checked that the extension does not already exist on the [website extensions list](https://tachiyomi.org/extensions/) or the app.
- [X] I have checked that the extension does not already exist by searching the [GitHub repository](https://github.com/tachiyomiorg/tachiyomi-extensions/) and verified it does not appear in the code base.
- [X] I will fill out all of the requested information in this form.

**Comment by stevenyomi** [comment]
> Basically the same structure as the shonen jump section so Viz might be able to be made into a multisrc

If it's the same site I think it's easier to make it a factory source.

**Comment by Notadev-justhelping** [comment]
Not sure if I should make a new issue about this but some viz manga don't seem to show up through the Viz Shonen Jump extension. For example, this one
`https://www.viz.com/vizmanga/chapters/call-of-the-night`
I can read it on the site, but it doesn't show up at all through the tachiyomi extension.

**Comment by AntsyLich** [comment]
@Notadev-justhelping so you read the title and proceed to comment without reading anything else?

**Comment by strider2112** [comment]
So what's going on with this issue? It seems like it gets automatically closed whenever someone opens it
